recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ source:
         - install_scripts.patch
 
 build:
-    number: 1
+    number: 2
     features:
```

Comment:
@msarahan As in, instead of features, add the actual `vs_runtime` in the build requirements?